src/js/index.js

```diff
@@ -24,6 +26,7 @@ class FHCookieGuard {
                 onRevoke: null
             }
         }, options, this.cookieAlert ? this.cookieAlert.dataset : {}]);
+        this._currentRequestCount = Cookie.get(this.options.autoAcceptCookieConsentName);
```

Comment:
`parseInt(Cookie.get(this.options.autoAcceptCookieConsentName)) || 0`

Dan is de variabele ook gelijk op `0` geïnitialiseerd.